app/uk/gov/hmrc/essttpstubs/services/TTP.scala

```diff
@@ -16,19 +16,19 @@
 
 package uk.gov.hmrc.essttpstubs.services
 
+import play.api.libs.json.Json
 import uk.gov.hmrc.essttpstubs.model._
 import uk.gov.hmrc.essttpstubs.services.EligibilityService.EligibilityError
-import uk.gov.hmrc.essttpstubs.services.TTP.asError
+import uk.gov.hmrc.essttpstubs.ttp.model.TTPEligibilityData
 
-import java.time.LocalDate
 
-case class TTP(items: Map[TaxID, Either[EligibilityError, OverduePayments]]){
+case class TTP(items: Map[TaxID, TTPEligibilityData]){
 
-  def mapError(taxID: TaxID, error: EligibilityError): TTP = TTP(items + (taxID -> asError(error)))
+  // def mapError(taxID: TaxID, error: EligibilityError): TTP = TTP(items + (taxID -> asError(error)))
 
-  def mapFinancialData(taxID: TaxID, data: OverduePayments): TTP = TTP(items + (taxID -> Right(data)))
+  // def mapFinancialData(taxID: TaxID, data: OverduePayments): TTP = TTP(items + (taxID -> Right(data)))
 
-  def eligibilityData(taxID: TaxID): Either[EligibilityError,OverduePayments] = items.getOrElse(taxID, Right(TTP.Default))
+  def eligibilityData(taxID: TaxID): TTPEligibilityData = items.getOrElse(taxID, TTP.Default)
```

Comment:
Could we please name types and values as `taxId` and `TtpElibilityData`.
This is how we do it in other payments projects. And it's easier to read it (see made up example: `VATIDXML` vs `VatIdXml` )

Reply:
done